src/mscorlib/model.xml

```diff
@@ -8608,6 +8608,14 @@
       <Member Name="GetHashCode" />
       <Member Name="ToString" />
       <Member MemberType="Property" Name="Item1" />
+      <Member Status="ImplRoot" Name="System.Collections.IStructuralComparable.CompareTo(System.Object,System.Collections.IComparer)" />
```

Comment:
I've spot checked the `S.P.Corelib.dll` and the new APIs appear there.
I've left the interface implementations in the `model.xml`. Let me know if they need to be removed, or ok to leave. Thanks